dev-docs/bidders/smartx.md

```diff
@@ -0,0 +1,43 @@
+|----------------------|----------|---------------------------------------------------------------------------------------------------------------------------------------------------------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|-------------|
+| `tagId`              | required | A unique ID.                                                                                                                                                      | `'Nu68JuOWAvrbzoyrOR9a7A'`                                                                                                                                                                            | `string`    |
+| `publisherId`        | required | A unique PublisherID.                                                                                                                                             | `'publisher_Id'`                                                                                                                                                                                      | `string`    |
+| `siteId`             | required | A unique SiteID.                                                                                                                                                  | `'site_id'`                                                                                                                                                                                           | `string`    |
+| `bidfloor`           | required | Value of Bidfloor.                                                                                                                                                |                              `0.3`                                                                                                                                                                                                 | `float`     |
+| `bidfloorcur`        | required | Used Currency. (e.g. EUR, USD etc.)                                                                                                                               | `'EUR'`                                                                                                                                                                                               | `string`    |
+| `context`            | required | Token that describes which context to play: 'instream' or 'outstream'                                                                                             | `'outstream'`                                                                                                                                                                                         | `string`    |
```

Comment:
It's not required as far as I can see in the implementation. Maybe this can be removed completely and only the `mediaType.video.context` parameter is being used.

Reply:
Changed to optional